docs/release_notes/v-1.6.md

```diff
@@ -0,0 +1,120 @@
+
+![A stream that returns a zip file with a CSV inside](assets/1-6-decoders.png)
+
+### New API endpoints to create custom connectors from YAML or Docker images
+
+Airbyte's public APIs and Terraform provider have new endpoints and resources you can use to create and update custom connectors using a Connector Builder YAML manifest or Docker image. The following new endpoints are available.
+
+For YAML source definitions:
+
+- `GET` - /workspaces/`{workspaceId}`/definitions/declarative_sources/ - List YAML source definitions
+- `POST` - /workspaces/`{workspaceId}`/definitions/declarative_sources/ - Create YAML source definitions
+- `GET/` - /workspaces/`{workspaceId}`/definitions/declarative_sources/`{definitionId}` - Get a specific YAML source definition
+- `PUT` - /workspaces/`{workspaceId}`/definitions/declarative_sources/`{definitionId}` - Update a specific YAML source definition
+- `DELETE` - /workspaces/`{workspaceId}`/definitions/declarative_sources/`{definitionId}` - Delete a specific YAML source definition
+
+For Docker source definitions:
+
+- `GET` - /workspaces/`{workspaceId}`/definitions/sources/ - List source definitions
+- `POST` - /workspaces/`{workspaceId}`/definitions/sources/ - Create source definitions
+- `GET` - /workspaces/`{workspaceId}`/definitions/sources/`{definitionId}` - Get a specific source definition
+- `PUT` - /workspaces/`{workspaceId}`/definitions/sources/`{definitionId}` - Update a specific source definition
+- `DELETE` - /workspaces/`{workspaceId}`/definitions/sources/`{definitionId}` - Delete a specific source definition
+
+For destination definitions:
+
+- `GET` /workspaces/`{workspaceId}`/definitions/destinations - List destination definitions
+- `POST` /workspaces/`{workspaceId}`/definitions/destinations - Create destination definitions
+- `GET` /workspaces/`{workspaceId}`/definitions/destinations/`{definitionId}` - Get a specific destination definition
+- `PUT` /workspaces/`{workspaceId}`/definitions/destinations/`{definitionId}` - Update a specific destination definition
+- `DELETE` /workspaces/`{workspaceId}`/definitions/destinations/`{definitionId}` - Delete a specific destination definition
+
+You can't use these endpoints to modify Airbyte's public connector definitions. If you're using the Cloud version of Airbyte, you can only create or modify custom connectors using a YAML manifest.
+
+[**API reference docs >**](https://reference.airbyte.com/reference/getting-started#/)
+
+[**Terraform reference docs >**](https://registry.terraform.io/providers/airbytehq/airbyte/latest/docs)
+
+## 🚀 Self-Managed Enterprise changes
+
+These changes bring new capabilities to Airbyte's Self-Managed Enterprise customers.
+
+:::danger Breaking Change
+Before upgrading from version 1.5.1 or earlier, you must upgrade your service account permissions. If you don't, Airbyte doesn't permit you to begin the upgrade. [Learn more](../enterprise-setup/upgrade-service-account.md).
+:::
+
+### Deploy Airbyte in multiple regions
```

Comment:
Can we use `across`? Or `Multi-Region Airbyte Deployments`? 

You can deploy Airbyte in multiple regions today with separate deployments